contracts/registry/PocketRegistryDelegate.sol

```diff
@@ -28,131 +29,95 @@ contract PocketRegistryDelegate {
-      if (nodeRecords[_nodeAddress].owner == msg.sender) {
-        nodeRecords[_nodeAddress].owner = newOwner;
-        } else {
-          revert();
-        }
-      }
-
-      // Tells whether a given Node key is registered.
-      function isRegisteredNode(address _nodeAddress) returns(bool) {
-        return nodeRecords[key].time != 0;
-      }
-
-      function getNodeRecordAtIndex(uint rindex) returns(address _nodeAddress) {
-        Record record = nodeRecords[nodeRecordsIndex[rindex]];
-        _nodeAddress = nodeRecordsIndex[rindex];
-        owner = record.owner;
-        time = record.time;
-        url = record.url;
-      }
-
-      // Returns the owner of the given record. The owner could also be get
-      // by using the function getRecord but in that case all record attributes
-      // are returned.
-      function getNodeOwner(address _nodeAddress) returns(address owner) {
-        return nodeRecords[_nodeAddress].owner;
-      }
-
-      // Returns the registration time of the given record. The time could also
-      // be get by using the function getRecord but in that case all record attributes
-      // are returned.
-      function getNodeTime(address _nodeAddress) returns(uint time) {
-        return nodeRecords[_nodeAddress].time;
-      }
-
-      // Registry owner can use this function to withdraw any value owned by
-      // the registry.
-      function withdraw(address to, uint value) onlyOwner {
-        if (!to.send(value)) revert();
-      }
-
-      function kill() onlyOwner {
-        suicide(owner);
-      }
-
-      // Get list of nodes that are currently relaying transactions
-      function getLiveNodes() constant returns (address[]) {
-        return registeredNodes;
-      }
-
-      function getCurrentNode () constant returns (address) {
-        return userNode[msg.sender];
-      }
-
-      function setTokenAddress(address _tokenAddress) {
-        assert(owner == msg.sender);
-        tokenAddress = _tokenAddress;
-      }
-
-      function setNodeDelegateAddress(address _nodeDelegateAddress) {
-        assert(owner == msg.sender);
-        nodeDelegateAddress = _nodeDelegateAddress;
-      }
+    // Returns the owner of the given record. The owner could also be get
+    // by using the function getRecord but in that case all record attributes
+    // are returned.
+    function getNodeOwner(address _nodeAddress) returns(address owner) {
+      require(_nodeAddress);
+      return getOwner(_nodeAddress);
+    }
+
+    // Returns the registration time of the given record. The time could also
+    // be get by using the function getRecord but in that case all record attributes
+    // are returned.
+    function getNodeTime(address _nodeAddress) returns(uint time) {
+      require(_nodeAddress);
+      return getTime(_nodeAddress);
+    }
+
+    // Registry owner can use this function to withdraw any value owned by
+    // the registry.
+    function withdraw(address to, uint value) onlyOwner {
```

Comment:
Remove this, we need to figure out the withdraw strategy for all of our contracts